dockstore-client/src/main/java/io/dockstore/client/cli/JCommanderUtility.java

```diff
@@ -63,6 +69,39 @@ public static void printJCommanderHelpLaunch(JCommander jc, String programName,
         printJCommanderHelpFooter();
     }
 
+    public static boolean wasErrorDueToUnknownParamter(String errorMsg) {
```

Comment:
I couldn't find a way to determine if `JCommander parse` failed because it didn't understand a command or because it was missing a required command. As they both throw the same exception. So I created this method to read the error message to determine if it failed because it didn't understand the command.

I don't love this approach, but I couldn't find a better approach.

The error message is always of the form,
```
Was passed main parameter 'unknown_command' but no main parameter was defined in your arg class
```

Reply:
Stick a TODO in here, it sounds like something that may be looked at during a version upgrade